### PR TITLE
Fixes #186: support for themed email is added

### LINKED ISSUE #186: send a  themed HTML email
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

Now we sending normal HTML email after documentation generation
### Expected behaviour

Instead of sending normal HTML email. send themed HTML email after documentation generation, 

### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

I have made a themed email to send after documentation generation
## Related Issue

close #186 
## Screenshots (if appropriate):
email

## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by imujjwal96** [comment]
Won't it be possible to implement this using jade itself?

**Comment by sch00lb0y** [comment]
I have converted from ejs to jade. @imujjwal96 please check

File: template/email.jade [hunk 1]
```diff
@@ -0,0 +1,146 @@
+doctype html
```
Comment (imujjwal96) [inline template/email.jade]: Move this to views/template/email.jade

**Comment by imujjwal96** [comment]
Also, can you show a demo?

**Comment by sch00lb0y** [comment]
email_previes
@imujjwal96 working screenshot

File: template/email.jade (hunk 1 above)
Comment (sch00lb0y) [inline template/email.jade]: I have moved as you suggested

**Pull request merged**